docs/publishers/cli.md

```diff
@@ -1,69 +1,86 @@
+The data is a command-line tool aimed to help publishers to prepare and upload data to the DataHub. With data you will be able to:
 
+* Publish Data Package to DataHub
+* Get Data Package from DataHub
+* Remove uploaded Data Package from DataHub
+* Get information about particular Data Package
+* Normalize Data Package according to the specs
 * Validate your data to ensure its quality
-* Publish Data Package
-* Tag uploaded Data Package to create historical snapshot
-* Remove uploaded Data Package that is no longer needed
+* Set up configuration file in order to publish
 
 ## Installation
 
-You can install unstable version directly from the code repository:
+### Installing binaries without npm
 
+On the [releases](https://github.com/datopian/datahub-cli/releases) page, you can download pre-built binaries for MacOS and LinuxOS x64. You need to put pre-built binary in bin directory, your path eg: `/usr/local/bin/`.
+ 
 ```bash
-pip install git+https://github.com/frictionlessdata/dpm-py.git
+mv path/to/data-{os-distribution} /usr/local/bin/data
 ```
 
+### Installing from npm
+
+You can also install it from [npm]() as follows:
+`npm install -g data`
+
 ## Commands
 
 You can see the latest commands and get help by doing:
 
 ```bash
-dpm --help
+data --help
 ```
 
 You will see output like this:
 
 ```bash
-Usage: dpm [OPTIONS] COMMAND [ARGS]...
+❒ data [options] <command> <args>
+
+Commands:
+  DataHub:
+    push        [path]        Push data to the DataHub
+    get         [pkg-id]      Get data from DataHub
+    purge       [owner/name]  Permanently deletes data from DataHub
+  Data Package specific:
+    info        [pkg-id]      Get info on data
+    normalize                 Normalize datapackage.json
+    validate                  Validate Data Package structure
+
+  Administrative:
+    config                    Set up configuration
+    help        [cmd]         Show help on cmd
 
 Options:
-  --version      Show the version and exit.
-  --config TEXT  Use custom config file. Default /home/u1/.dpm/config
-  --debug        Show debug messages
-  --help         Show this message and exit.
+-h, --help              Output usage information
+-v, --version           Output the version
 
-Commands:
-  configure     Update configuration options.
-  datavalidate  Validate csv file data, given its path.
-  delete        Delete Data Package from the registry server.
-  publish       Publish Data Package to the registry server.
-  purge         Purge Data Package from the registry server.
-  tag           Tag Data Package on the server.
-  undelete      Undelete Data Package from the registry...
-  validate      Validate Data Package in the current dir.
 ```
 
 ## Configuration
 
-Dpm can be configured using `dpm configure` command. It will ask you
+Data can be configured using `data config[ure]` command. It will ask you
 to provide username, access_token and server address of DataHub.
```

Comment:
It will ask you to provide a username, secretToken, server and bitStore addresses of DataHub.